atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2080,7 +2080,7 @@ v8::Local<v8::Value> WebContents::GetLastWebPreferences(
 }
 
 bool WebContents::IsRemoteModuleEnabled() const {
-  if (web_contents()->GetVisibleURL().SchemeIs("chrome-devtools")) {
+  if (web_contents()->GetVisibleURL().SchemeIs("devtools")) {
```

Comment:
Can you replace it all with `kChromeDevToolsScheme` , easier to maintain.

Reply:
Makes sense, I just followed what we were already doing.
We can probably make that change in a separate PR for other urls as well, like `chrome-extension`.